04-操作系统补丁/SSDT-OC-XOSI.dsl

```diff
@@ -3,6 +3,10 @@
 // Find:     4F534944
 // Replace:  58534944
 //
+// In config ACPI, OSID to XSID
```

Comment:
Hi, this looks good for me! But is this line should be OSIF to XSIF?

Reply:
Yeah, a typo error!